internal/util/schemaupdate/schemaupdate.go

```diff
@@ -19,18 +19,26 @@ import (
 // ValidateSchemaUpdate validates that the new json schema doesn't break
 // profiles using this rule type
 func ValidateSchemaUpdate(oldRuleSchema *structpb.Struct, newRuleSchema *structpb.Struct) error {
```

Comment:
Given the lovely new functions in schemavalidate for conversion, would it make sense to file a cleanup ticket to make these take a pair of `jsonschema.Schema`?

Reply:
https://github.com/mindersec/minder/issues/5015